boards/shields/nxp_m2_wifi_bt/doc/index.rst

```diff
@@ -0,0 +1,60 @@
+.. _nxp_m2_wifi_bt:
+
+NXP M.2 Wi-Fi and BT Shield
+###########################
+
+Overview
+********
+
+This Zephyr shield is tested with the following M.2 modules and hardware for Wi-Fi and Bluetooth applications:
+- Embedded Artist 1XK module - uses Murata 1XK radio module with NXP IW416 chipset
+- Embedded Artist 2EL module - uses Murata 2EL radio module with NXP IW612 chipset
+
+More information about supported chipsets, radio modules and M.2 modules can be found in below links,
+
+`IW612 NXP Chipset <https://www.nxp.com/products/IW612>`_
+`IW416 NXP Chipset <https://www.nxp.com/products/IW416>`_
+`2EL Murata Radio Module <https://www.murata.com/en-us/products/connectivitymodule/wi-fi-bluetooth/overview/lineup/type2el>`_
+`1XK Murata Radio Module  <https://www.murata.com/en-us/products/connectivitymodule/wi-fi-bluetooth/overview/lineup/type1xk>`_
+`1XK Embedded Artist Module <https://www.embeddedartists.com/products/1xk-m-2-module>`_
+`2EL Embedded Artist Module <https://www.embeddedartists.com/products/2el-m-2-module>`_
```

Comment:
Make a proper bullet list - see here how it looks like currently https://builds.zephyrproject.io/zephyr/pr/85639/docs/boards/shields/nxp_m2_wifi_bt/doc/index.html